### PR TITLE
Updated documentation related to the association with quantities.

### PR DESCRIPTION
**Pull request by felix-couprie**
This is the very first commit of the updated documentation related to the association with quantities.
It impacts 3 different pages:
- create association types
- manage your associations
- variant products

I've added some screenshots, updated the text and rearranged few paragraphs.

### CODE REVIEW

File: content/md/first-steps/manage-type-association.md [hunk 1]
```diff
@@ -33,19 +34,19 @@ Read our [Association types](manage-your-association-types.html) section to find
...
-You must have the permission to create an association type from the PIM.
+You must have the permission to create an association type from the PIM. Association type must contain at least one letter.
 :::
 
 ## Create a 1-way association type
 
 To create a 1-way association type:
 1.  Click on `Create association type`
-1.  Fill in the code of the association type you want to create
+1.  Select the code of the association type you want to create
```
Comment (SamirBoulil) [inline content/md/first-steps/manage-type-association.md:44]: Fill in is better no ?

File: content/md/products-super-power/product-associations.md [hunk 2]
```diff
@@ -11,8 +11,10 @@ related: manage-your-association-types
 
 A product may be associated with none or several other products or product groups. This type of link has a direction: 1-way or 2-way. Please read our [Association Types documentation](manage-your-association-types.html#Overview) for more information.
 
-::: info
-These types of associations enables you to define relationships, for example like an alternative product, a substitution between products, or even a cross-sell/up-sell option.
+These types of associations enables you to define relationships, for example like an alternative product, a substitution between products, a cross-sell/up-sell option or even a product set composed with different quantities of products.
```
Comment (SamirBoulil) [inline content/md/products-super-power/product-associations.md:14]:
```suggestion
These types of associations enable you to define relationships, for example like an alternative product, a substitution between products, a cross-sell/up-sell option or even a product set composed with different quantities of products.
```

File: content/md/products-super-power/product-associations.md [hunk 3]
```diff
@@ -71,6 +74,30 @@ To add associations, there is a picker to select products or [product models](wh
 A product can also be associated to a [product model](what-about-products-variants.html#what-is-a-product-model).
 :::
 
+## 2nd way: add associations to a product using the association type with quantities
```
Comment (SamirBoulil) [inline content/md/products-super-power/product-associations.md:77]:
```suggestion
## 2nd way: add associations to a product using an association type with quantities
```

**Comment by SamirBoulil** [comment]
Make sure you have it reviewed by @camillebourjade (if she's back at the office)

File: content/md/first-steps/manage-type-association.md [hunk 4]
```diff
@@ -60,13 +61,30 @@ To create a 2-way association type:
...
 :::warning
 The 2-way association property cannot be updated once the association type is created.
 :::
 
+## Create an association type with quantities
+
+To create an association type with quantities:
+1.  Click on `Create association type`
+1.  Select the code of the association type you want to create
+1.  Set the "Association with quantity" button to "Yes"
```
Comment (juliensnz) [inline content/md/first-steps/manage-type-association.md:76]:
```suggestion
1.  Set the "Association with quantity" switch to "Yes"
```

File: content/md/first-steps/manage-type-association.md [hunk 5]
```diff
@@ -78,7 +96,7 @@ To edit an association type:
 
 ::: warning
 You cannot change the code once you created the association type.    
-Neither will you be able to update the 2-way association. 
+Neither will you be able to update the 2-way association nor the association with quantities.
```
Comment (juliensnz) [inline content/md/first-steps/manage-type-association.md:99]:
```suggestion
Neither will you be able to change it to a 2-way association nor an association with quantities.
```

**Comment by jmleroux** [comment]
related to PIM-9331

File: content/md/products-super-power/product-associations.md [hunk 6]
```diff
@@ -71,6 +74,30 @@ To add associations, there is a picker to select products or [product models](wh
...
+
+1.  Create a product model that will be the base for your product set
+1.  Open the product form
+1.  Click on the `Associations` tab
+1.  Select the association type with quantities
+  ![Select an association type](../img/Products_Products_Associations_Select_Association_type.png)
+1.  Click on the `Add associations` button
+1.  The picker to select product or product models opens
+  ![Associations products picker](../img/Products_Products_Associations_Picker.png)
+1.  Filter on the categories, on the label or identifier or add filters to find the products or product models to associate
```
Comment (camillebourjade) [inline content/md/products-super-power/product-associations.md:87]:
```suggestion
1.  Filter on the categories, on the label or identifier or add filters to find the products or product models you want to associate
```

File: content/md/products-super-power/product-associations.md [hunk 7]
```diff
@@ -71,6 +74,30 @@ To add associations, there is a picker to select products or [product models](wh
...
+1.  Tick in the grid the products or product models to associate, they are displayed in the basket
```
Comment (camillebourjade) [inline content/md/products-super-power/product-associations.md:88]:
```suggestion
1. In the grid, tick the products or product models to associate. They are displayed in the basket
```

File: content/md/products-super-power/product-associations.md [hunk 8]
```diff
@@ -71,6 +74,30 @@ To add associations, there is a picker to select products or [product models](wh
...
+  ![Tick products to associate](../img/Products_Products_Associations_Basket.png)
+1.  Click on the `Confirm` button
```
Comment (camillebourjade) [inline content/md/products-super-power/product-associations.md:90]:
```suggestion
1.  Click on `Confirm`
```

File: content/md/products-super-power/product-associations.md [hunk 9]
```diff
@@ -71,6 +74,30 @@ To add associations, there is a picker to select products or [product models](wh
...
+1.  The products or product models you associate to the current product are displayed in the `Associations` tab
+  ![View the associated products](../img/Products_Products_Associations_associations_results.png)
+1.  Adjust the quantities for each associated product/product model
+  ![Adjust associated products quantities](../img/Products_Products_Associations_Quantities.png)
+1.  Click on the `Save` button
```
Comment (camillebourjade) [inline content/md/products-super-power/product-associations.md:95]:
```suggestion
1.  Click on `Save`
```

File: content/md/first-steps/manage-type-association.md [hunk 10]
```diff
@@ -45,28 +46,45 @@ To create a 1-way association type:
...
 :::warning
 The 2-way association property cannot be updated once the association type is created.
 :::
 
+## Create an association type with quantities
+
+To create an association type with quantities:
+1.  Click on `Create association type`
+1.  Select the code of the association type you want to create
+1.  Set the "Association with quantity" switch to "Yes"
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/md/first-steps/manage-type-association.md:76]: Switch :)

[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/md/first-steps/manage-type-association.md:76]: "switch"!

File: content/md/first-steps/manage-type-association.md [hunk 11]
```diff
@@ -12,9 +12,10 @@ Associations types are used to link products together.
 
 For example, you can link a product to its accessories. The code for this association type could be “ACCESSORIES”.
 
-There are two different association types:
+There are three different association types:
 *   The 1-way association (product A is associated with product B).
 *   The 2-way association (when product A is associated with product B, then product B will be automatically associated with product A).
+*   The association with quantities (product A is associated with product B for a quantity of 2 and with product C with a quantity of 3)
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/md/first-steps/manage-type-association.md]: changed!

**Pull request merged**